src/components/CodePreview.tsx

```diff
@@ -14,6 +16,11 @@ interface CodePreviewProps {
 
 export function CodePreview({ code, raw }: CodePreviewProps) {
   const [hasCopiedToClipboard, setCopiedToClipboard] = useState(false)
+  const [isExplorerOpen] = useAtom(isExplorerOpenAtom)
```

Comment:
```suggestion
  const isExplorerOpen = useAtomValue(isExplorerOpenAtom)
```
This way, the component will not keep track of the setter updates, only the value.